auth0/testdata/recordings/TestAccConnectionAD.yaml

```diff
@@ -3,17 +3,17 @@ version: 1
 interactions:
 - request:
     body: |
-      {"name":"Acceptance-Test-AD-TestAccConnectionAD","strategy":"ad","show_as_button":true,"options":{"tenant_domain":"example.com","domain_aliases":["api.example.com","example.com"],"ips":["192.168.1.1","192.168.1.2"],"brute_force_protection":true,"set_user_root_attributes":"on_each_login","non_persistent_attrs":["ethnicity","gender"]}}
+      {"name":"Acceptance-Test-AD-TestAccConnectionAD","strategy":"ad","show_as_button":true,"options":{"tenant_domain":"example.com","domain_aliases":["api.example.com","example.com"],"ips":["192.168.1.1","192.168.1.2"],"brute_force_protection":true,"set_user_root_attributes":"on_each_login","non_persistent_attrs":["ethnicity","gender"],"upstream_params":{"screen_name":{"alias":"login_hint"}}}}
     form: {}
     headers:
       Content-Type:
       - application/json
       User-Agent:
-      - Go-Auth0-SDK/0.6.3
+      - Go-Auth0-SDK/latest
     url: https://terraform-provider-auth0-dev.eu.auth0.com/api/v2/connections
     method: POST
   response:
-    body: '{"id":"con_aBhf6n2pgXp17087","options":{"tenant_domain":"example.com","domain_aliases":["api.example.com","example.com"],"ips":["192.168.1.1","192.168.1.2"],"brute_force_protection":true,"set_user_root_attributes":"on_each_login","non_persistent_attrs":["ethnicity","gender"]},"strategy":"ad","name":"Acceptance-Test-AD-TestAccConnectionAD","provisioning_ticket_url":"https://terraform-provider-auth0-dev.eu.auth0.com/p/ad/fOQMg2YofuuI6jL3sZoBYuks3jWC4lCO","is_domain_connection":false,"show_as_button":true,"enabled_clients":[],"realms":["Acceptance-Test-AD-TestAccConnectionAD"]}'
+    body: '{"id":"con_pFtc8ZCVuJZnSoEP","options":{"tenant_domain":"example.com","domain_aliases":["api.example.com","example.com"],"ips":["192.168.1.1","192.168.1.2"],"brute_force_protection":true,"set_user_root_attributes":"on_each_login","non_persistent_attrs":["ethnicity","gender"],"upstream_params":{"screen_name":{"alias":"login_hint"}}},"strategy":"ad","name":"Acceptance-Test-AD-TestAccConnectionAD","provisioning_ticket_url":"https://sergiu-dev.eu.auth0.com/p/ad/dNtFUMuCIeEh3tRmXaVt0zqPB0JOLR7H","is_domain_connection":false,"show_as_button":true,"enabled_clients":[],"realms":["Acceptance-Test-AD-TestAccConnectionAD"]}'
```

Comment:
This will be re-recorded again against `terraform-provider-auth0-dev.eu.auth0.com` using a specific go-auth0 SDK version before merging, right?

Reply:
Yes, absolutely:) for now it's just to make sure tests pass and we don't need to keep modifying the API on the go-auth0 PR counterpart.